client/src/main/scala/org/apache/celeborn/client/LifecycleManager.scala

```diff
@@ -379,7 +379,7 @@ class LifecycleManager(val appUniqueId: String, val conf: CelebornConf) extends
         }
         causes.add(Utils.toStatusCode(info.getStatus))
       }
-      logWarning(s"Received Revive request, number of partitions ${partitionIds.size()}")
+      logDebug(s"Received Revive request, number of partitions ${partitionIds.size()}")
```

Comment:
IMO, changing this log to debug is too aggressive. These logs are important to show the running status of a spark app. Maybe changing it to an info level would be better.

Reply:
@FMX These logs gets print very often – example for a small test application i am running driver logs have around 40K lines of this exact log – which is not very useful for any debugging. This code flow will anyway trigger the ChangePartitionManager flow which will print the log consolidating these revive request.
```
ChangePartitionManager: Batch handle change partition for [0-0-0,0-3-0]
```

Reply:
Sounds reasonable

Reply:
@s0nskar, +1. Too many revive log is unnecessary for application.